### PR TITLE
GH-280 Added support for manager roles in artifactory_group

### LINKED ISSUE #280: Add support for manager roles in artifactory_group
**Is your feature request related to a problem? Please describe.**
I'm trying to provision a group of XRay admins with Terraform, but I'm unable to grant the "watchManager", "policyManager", and "reportsManager" roles. These are exposed via the API but not in the Terraform provider.

**Describe the solution you'd like**
Allow us to specify these roles in the artifactory_group resource. These are mentioned here as not yet exposed to the user: https://github.com/jfrog/terraform-provider-artifactory/blob/master/pkg/artifactory/resource_artifactory_group.go#L224-L226

**Describe alternatives you've considered**
I could provision this group via UI or API, but the rest of our permission structure is managed with terraform, so would prefer that to be consistent.

### PR DESCRIPTION
**Pull request by maheshjfrog**
Fixes #280

### CODE REVIEW

**Review by chb0github — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+## 2.15.3 (Feb, 4, 2022)
+
+IMPROVEMENTS:
+
+* Added support for manager roles in artifactory_group resource [GH-308]
```
Comment (alexhung) [inline CHANGELOG.md:5]: Please include the resource name per guideline, see 2.15.2 for example.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
+## 2.15.3 (Feb, 4, 2022)
```
Comment (alexhung) [inline CHANGELOG.md:1]: This should be 2.16.0

**Review by alexhung — CHANGES_REQUESTED** [review]
LGTM apart from the CHANGELOG formatting

File: CHANGELOG.md (hunk 1 above)
Comment (maheshjfrog) [reply CHANGELOG.md]: done

**Review by alexhung — APPROVED** [review]

**Pull request merged**